ci-operator/config/openshift/cluster-authentication-operator/openshift-cluster-authentication-operator-master.yaml

```diff
@@ -55,6 +55,11 @@ tests:
   steps:
     cluster_profile: aws
     workflow: origin-e2e-aws
+- artifact_dir: /tmp/artifacts
+  as: e2e-aws-kubeadmin-login
+  commands: "openshift-tests run all --run='\\[Feature:BootstrapUser\\]' -o /tmp/artifacts/e2e.log --junit-dir /tmp/artifacts/junit"
```

Comment:
What is the reason to run it separately and not as part of the regular e2e suite? If there is a reason I suppose it's be better to create it's own suite in e2e command.